Code/Common/src/sitkPimpleImageBase.hxx

```diff
@@ -98,7 +98,7 @@ namespace itk
 
         dup->SetInputImage( this->m_Image );
         dup->Update();
-        ImagePointer output = dup->GetOutput();
+        ImagePointer output = dup->GetModifiableOutput();
```

Comment:
@hjmjohnson 

The remaining interface to me seems odd. It the `ImageDuplicator` trying to be like a filter with `Update` and `GetOutput`?

This class seems stuck in the middle for its expected behavior, and I am not sure there is a clear reference for expected behavior for this "non-filter". There are the `Calculators` but the do not seem like a similar model.